help/en/package/jmri/jmrit/simpleclock/SimpleClockFrame.shtml

```diff
@@ -135,11 +140,17 @@
       Fast clock settings are stored as part of your configuration
       in the same file as your panels (if you have any). The safest
       and easiest way to ensure that all your configuration items
-      and panels are saved is to save using <b>Store panels...</b>.
-      <b>Store panels...</b>may be found in the <b>Panels</b> menu
-      and the <b>File</b> menu of Layout Editor and Panel Editor.
-      (<b>Store panels...</b> may be used even if you don't have
-      any panels.) If you want to automatically load your panel
+      and panels are saved is to save using <b>Store Configuration
+      and panels...</b>. (<b>Store panels...</b> may be used even
+      if you don't have any panels.)
+      <a href="images/FastClockSetupSave.png">
+      <img src="images/FastClockSetupSave.png" width="333" height="140"
+      alt="Fast Clock Setup config pane" align="right"></a>
+      <b>Store</b> options may be found in the <b>File</b> menu
+      while working on the Fast Clock Setup. A prompt will give
+      you the option to save either form when you close the
+      Fast Clock Setup by clicking Apply.</p>
```

Comment:
And again, setup.

Reply:
Let's go for setup (as noun), see next commit